Update user-tokens.md (#5575)
dbt-labs/docs.getdbt.com: website/docs/docs/dbt-cloud-apis/user-tokens.md

:::note Announcement

> [User API keys](#user-tokens) are being replaced by [account-scoped Personal Access Tokens(PATs)](#account-scoped-personal-access-tokens). 

Reviewer: We should only capitalize products (dbt Explorer, dbt Cloud, but not features like column-level lineage), Acronyms and proper nouns. I will add this to the style guide so it's better communicated!
```suggestion
[User API keys](#user-tokens) are being replaced by [account-scoped personal access tokens(PATs)](#account-scoped-personal-access-tokens). 
```

We recommend rotating your existing user API keys with PATs. There are no deprecation plans for user API keys at this time; we will give ample notice when that timeline has been determined. 

Cloud CLI config files do not need to be updated at this time. You will be notified when you need to re-download your configs.

The current user API key is located under **Personal Settings → API Key.** 

Please [contact support](/docs/dbt-support#dbt-cloud-support) with any questions or concerns.

:::

> ## Account-scoped Personal Access Tokens

Reviewer: @Millacy Heads up that we use [sentence case](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#Titles) in titles for a more conversational tone.
```suggestion
## Account-scoped personal access tokens
```

:::info New 

On Feb 7, 2024, we introduced a new type of token for individual users called Personal Access Tokens. Note that these differ from [Service Tokens or user API Keys](/docs/dbt-cloud-apis/authentication#types-of-api-access-tokens). 

> Before this release, User API keys were the only way to access dbt Cloud API on behalf of the user. 

Reviewer: This sentence feels very linked to the previous paragraph. Is there a reason you separated it from the paragraph?
```suggestion
Before this release, user API keys were the only way to access dbt Cloud API on behalf of the user. 
```

Reviewer 2: Added a break for readability, the large block felt difficult to parse:
<img width="826" alt="Screenshot 2024-05-29 at 9 57 21 PM" src="https://github.com/dbt-labs/docs.getdbt.com/assets/18703280/e56ace77-420c-4895-9f0b-667f3cd5e2ed">

Thoughts on keeping the first paragraph break below if i remove that 2nd paragraph break? Or would it be better to keep it as it was before?
<img width="826" alt="Screenshot 2024-05-29 at 10 02 21 PM" src="https://github.com/dbt-labs/docs.getdbt.com/assets/18703280/4a1f1b84-4cb7-408c-aa5c-9e68a47b390f">

These API Keys were user-specific and were not scoped to an account. To enhance the security of dbt Cloud, we are moving away from this model to account-specific tokens. 

:::

> Each dbt Cloud user with a [Developer license](https://docs.getdbt.com/docs/cloud/manage-access/seats-and-users) can create a new personal access token (PAT) to access the dbt Cloud API and dbt Cloud CLI. This token can execute queries against the dbt Cloud API on the user's behalf. To access dbt Cloud APIs and resources on behalf of the _account_, we recommend using Service Tokens instead. Learn more about [which token type you should use](/docs/dbt-cloud-apis/authentication#which-token-type-should-you-use) to understand the token differences.

Reviewer: ```suggestion
Each dbt Cloud user with a [Developer license](https://docs.getdbt.com/docs/cloud/manage-access/seats-and-users) can create a new personal access token (PAT) to access the dbt Cloud API and dbt Cloud CLI. This token can execute queries against the dbt Cloud API on the user's behalf. To access dbt Cloud APIs and resources on behalf of the _account_, we recommend using service tokens instead. Learn more about [which token type you should use](/docs/dbt-cloud-apis/authentication#which-token-type-should-you-use) to understand the token differences.
```

PATs inherit the permissions of the user that created them. For example, if a developer-licensed user with Project Admin role access to specific projects creates a PAT, the token will get the Project Admin role with access to the same projects as the user. These tokens are also account-specific, so if a user has access to more than one dbt Cloud account with the same email address, they need to create a unique PAT for each one of these accounts. 

> ### Migrate from user API keys to Personal Access Tokens

Reviewer: @Millacy Heads up that we use [sentence case](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#Titles) in title for a more conversational tone.
```suggestion
### Migrate from user API keys to personal access tokens
```

The migration to PATs is critical if you are using user API keys today. The current API key is located under **Personal Settings → API Key**.

 There are a few things to understand if you are using a user API key today: 

> * Personal Access Tokens are more secure. 

Reviewer: ```suggestion
* Personal access tokens are more secure. 
```

>     * To promote the least privilege and high-security assurance for your dbt Cloud accounts, we highly recommend moving to the new account-scoped Personal Access Tokens.

Reviewer: ```suggestion
    * To promote the least privilege and high-security assurance for your dbt Cloud accounts, we highly recommend moving to the new account-scoped personal access tokens.
```

:::note Announcement

[User API keys](#user-tokens) are being replaced by [account-scoped personal access tokens(PATs)](#account-scoped-personal-access-tokens). 

> We recommend rotating your existing user API keys with PATs. There are no deprecation plans for user API keys at this time; we will give ample notice when that timeline has been determined. 

Reviewer 3: ```suggestion
We recommend rotating your existing user API keys with PATs. At this time, there are no deprecation plans for user API keys; we will give ample notice when that timeline has been determined. 
```

:::

### Using the new personal access tokens
### Using the new Personal Access Tokens

Are you using a user API key today to access dbt Cloud APIs in any of your workflows? 

If not, you don’t have any action to take! 

> Are you using a user API key today to access dbt Cloud APIs in any of your workflows? If not, you don’t have any action to take. If you are using a user API key, please follow the instructions below. 

Reviewer: Nice catch!

:::

### Using the new Personal Access Tokens

Are you using a user API key today to access dbt Cloud APIs in any of your workflows? 

If not, you don’t have any action to take! 

> If you are using a user API key, please follow the instructions below:

Reviewer 3: ```suggestion
If you are using a user API key, please read the following instructions:
```
